### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Add readme with all information about configuration and running the script

Signed-off-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
 # Scripts
-Personal scripts used to automate various tasks
+
+Personal scripts used to automate various tasks.
+
+## Lab-grading
+
+Python script to automatically grade students.
+
+### Environment
```
Comment (teodutu) [inline README.md]: Move all the text below to its own `README.md` in the `lab-grading` folder and link the new `README.md` file here. This helps keep the global `README.md` as more scripts are added to this repo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+```
+pip3 install -r requirements.txt
+```
+
+### Configuration
+
+In order to run the script, you need to configure some files in `lab-grading`.
+
+Update `course-registers` file and add the ID for every course you want to use.
```
Comment (teodutu) [inline README.md]:
```suggestion
Update the `course-registers.json` file and replace the strings `"TODO"` with the IDs of the registers from the courses you teach.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+You can find the ID in the URL of the spreadsheet. More inforations can be found [here](https://developers.google.com/sheets/api/guides/concepts).
```
Comment (teodutu) [inline README.md]:
```suggestion
You can find the ID in the URL of the spreadsheet.
More information can be found [here](https://developers.google.com/sheets/api/guides/concepts).
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,48 @@
...
+
+To interact with Google API you need to generate a OAuth2.0 token.
+To do this for the first time we need to create a project on the Google Cloud platform.
+More information about how you can create a new project [here](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+After you create the project, you must generate an OAuth2.0 token following instructions
+from [here](https://support.google.com/cloud/answer/6158849?hl=en).
```
Comment (teodutu) [inline README.md]:
```suggestion
After you create the project, you must generate an OAuth2.0 token following instructions from [here](https://support.google.com/cloud/answer/6158849?hl=en).
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,48 @@
...
+First, you need to install a couple of packages that are listed in `requirements.txt` file.
+You can install them locally by running:
+
+```
+pip3 install -r requirements.txt
+```
+
+### Configuration
+
+In order to run the script, you need to configure some files in `lab-grading`.
```
Comment (teodutu) [inline README.md]:
```suggestion
In order to run the script, you need to configure some files in this folder.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,48 @@
...
+Update `course-registers` file and add the ID for every course you want to use.
+You can find the ID in the URL of the spreadsheet. More inforations can be found [here](https://developers.google.com/sheets/api/guides/concepts).
+
+To interact with Google API you need to generate a OAuth2.0 token.
+To do this for the first time we need to create a project on the Google Cloud platform.
+More information about how you can create a new project [here](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+After you create the project, you must generate an OAuth2.0 token following instructions
+from [here](https://support.google.com/cloud/answer/6158849?hl=en).
+Download the coresponding `json` format for it and add it to a new file named `credentials.json` in `lab-grading` folder.
```
Comment (teodutu) [inline README.md]:
```suggestion
Download the corresponding `json` format for it save it in a new file named `credentials.json`, placed in this folder.
```

**Review by teodutu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gabrielmocanu) [reply README.md]: Done.

**Review by teodutu — APPROVED** [review]

**Pull request merged**